modules/virt-using-singlereplica.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+[id="virt-using-singlereplica_{context}"]
+= Using SingleReplica mode
+
+Because the process of creating a cluster in `SingleReplica` mode or `HighlyAvailable` mode is determined at the time of installation, you do not need to configure anything specific to {VirtProductName} to use it on a single node. However; there are significant changes to the behavior of the cluster:
```

Comment:
Maybe reword the content to something like this:

```
If your cluster uses the `SingleReplica` mode, also known as Single Node OpenShift (SNO) (xref to OCP doc will go here), the following changes occur in {VirtProductName} behavior:

- {VirtProductName} components have only one replica and pod disruption budgets are not set.
- Live migration is not supported.
- Due to differences in storage behavior, some virtual machine templates are incompatible with `SingleReplica` mode. To ensure that the virtual machine is shut off in case of a node drain, you must not use the `evictionStrategy` field for any templates or virtual machines that use data volumes or storage profiles.
```

Reply:
After reading your edits, I revisited convos with a couple different folks. I incorporated some of your feedback, along with some different rewrites, and opted to better mirror the OCP page. Kept in some extra details devs felt was relevant.